src/field_theory/tower.lean

```diff
@@ -81,6 +81,40 @@ let ⟨c, hc⟩ := exists_is_basis_finset K A in
 by rw [findim_eq_card_basis hb, findim_eq_card_basis hc,
     findim_eq_card_basis (hb.smul hc), fintype.card_prod]
 
+/-- Given a tower `F : K : A`, any submodule of `A` over `K` is also a submodule of `A` over `F`. -/
+def submodule_tower_submodule (p : submodule K A) : submodule F A :=
```

Comment:
Isn't this just `submodule.restrict_scalars'`?

Reply:
You're right, I just wasn't aware of `submodule.restrict_scalars` (and possibly when the first version of this was written, `submodule.restrict_scalars` wasn't in mathlib yet). I've deleted this.